### ISSUE TITLE
Failing accessibility checks on "Find a resource..." and "Find an address" text contrast

### ISSUE THREAD

**Issue by samuelscandrett** [archesproject/arches#12521]
(spotted on Arches 7.5.x & 7.6.x)

On the Arches search page, the text contrast for "Find a resource..." and "Find an address" fails WCAG AA.

<img width="1785" height="228" alt="Image" src="https://github.com/user-attachments/assets/215e8284-509b-435a-a2af-0ee686a9c5f3" />

<br><br>

Although checks were run using Wave, these issues were discovered using a Chrome extension called [Silktide](https://chromewebstore.google.com/detail/silktide-accessibility-ch/mpobacholfblmnpnfbiomjkecoojakah).
See the accessibility checks for "Find a resource" below.  Note, "Find an address" resulted the same

<img width="919" height="509" alt="Image" src="https://github.com/user-attachments/assets/f037c8a7-5650-46c8-b2c1-0d6bcf032090" />
<img width="919" height="506" alt="Image" src="https://github.com/user-attachments/assets/e39f2581-1cf8-4228-af85-38d5b8065dd3" />

<br><br>

On [GLHER](https://glher.historicengland.org.uk/search), this text appears darker. 

<img width="1867" height="188" alt="Image" src="https://github.com/user-attachments/assets/98e4ac16-5234-4752-a49f-e290777f1b86" />

Perhaps this change was missed when pushing accessiblity changes back to core Arches.
